packages/flutter/test/material/drawer_test.dart

```diff
@@ -259,6 +259,60 @@ void main() {
     expect(state.isEndDrawerOpen, equals(false));
   });
 
+  testWidgets('Open/close drawer by dragging', (WidgetTester tester) async {
+    final ThemeData draggable = ThemeData(platform: TargetPlatform.android);
+    await tester.pumpWidget(
+      MaterialApp(
+        theme: draggable,
+        home: const Scaffold(drawer: Drawer()),
+      ),
+    );
+
+    final TestGesture gesture = await tester.createGesture();
+    final Finder finder = find.byType(Drawer);
+
+    double drawerPosition() {
+      expect(finder, findsOneWidget);
+      final RenderBox renderBox = tester.renderObject(finder);
+      return renderBox.localToGlobal(Offset.zero).dx;
+    }
+
+    // pointer down (drawer is closed)
```

Comment:
Very minor, consider capitalizing the first letter and adding a period at the end (same for the other comments).
```suggestion
    // Pointer down (drawer is closed).
```